FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -157,10 +157,20 @@ extension SetupInitialViewController {
         }
     }
 
+    private func getIdToken() async throws -> String {
+        let googleService = GoogleUserService(
+            currentUserEmail: user.email,
+            appDelegateGoogleSessionContainer: nil
+        )
```

Comment:
We probably could/should dependency-inject this through AppContext. I'll file another issue.